x-pack/plugins/security_solution/public/common/components/top_n/translations.ts

```diff
@@ -12,13 +12,13 @@ export const CLOSE = i18n.translate('xpack.securitySolution.topN.closeButtonLabe
 });
 
 export const ALL_EVENTS = i18n.translate('xpack.securitySolution.topN.allEventsSelectLabel', {
-  defaultMessage: 'All events',
+  defaultMessage: 'Alerts and events',
 });
 
 export const RAW_EVENTS = i18n.translate('xpack.securitySolution.topN.rawEventsSelectLabel', {
   defaultMessage: 'Raw events',
 });
 
 export const ALERT_EVENTS = i18n.translate('xpack.securitySolution.topN.alertEventsSelectLabel', {
-  defaultMessage: 'Detection Alerts',
+  defaultMessage: 'Detection alerts',
```

Comment:
Thanks for updating this from Title case to Sentence case per the [guidelines](https://elastic.github.io/eui/#/guidelines/writing/guidelines#capitalization) 🙏